### DISCUSSION TITLE
Templates for every project?

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#119] [Discuss a Topic]
Do we need to include the ISSUES_TEMPLATE and PULL_REQUESTS_TEMPLATE for every project we do from here on out? Or are those templates reserved for group projects?

**Comment by TrinityChristiana** [comment]
Those are not a requirement outside of your group projects. Really anything in that planning document isn't required outside of group projects unless the instructions say otherwise. 😃
